backend/FwLite/FwDataMiniLcmBridge/Api/RichTextMapping.cs

```diff
@@ -0,0 +1,486 @@
+﻿using System.Collections.Frozen;
+using System.Globalization;
+using System.Text;
+using MiniLcm.Models;
+using SIL.LCModel.Core.KernelInterfaces;
+using SIL.LCModel.Core.Text;
+using SIL.LCModel.Utils;
+
+namespace FwDataMiniLcmBridge.Api;
+
+public static class RichTextMapping
+{
+    public static readonly FrozenSet<FwTextPropType> IntProps =
+    [
+        FwTextPropType.ktptFontSize,
+        FwTextPropType.ktptOffset,
+        FwTextPropType.ktptFirstIndent,
+        FwTextPropType.ktptLeadingIndent,
+        FwTextPropType.ktptTrailingIndent,
+        FwTextPropType.ktptSpaceBefore,
+        FwTextPropType.ktptSpaceAfter,
+        FwTextPropType.ktptTabDef,
+        FwTextPropType.ktptLineHeight,
+        FwTextPropType.ktptRightToLeft,
+        FwTextPropType.ktptPadLeading,
+        FwTextPropType.ktptPadTrailing,
+        FwTextPropType.ktptMarginTop,
+        FwTextPropType.ktptPadTop,
+        FwTextPropType.ktptPadBottom,
+        FwTextPropType.ktptBorderTop,
+        FwTextPropType.ktptBorderBottom,
+        FwTextPropType.ktptBorderLeading,
+        FwTextPropType.ktptBorderTrailing,
+        FwTextPropType.ktptBulNumScheme,
+        FwTextPropType.ktptBulNumStartAt,
+        FwTextPropType.ktptDirectionDepth,
+        FwTextPropType.ktptKeepWithNext,
+        FwTextPropType.ktptKeepTogether,
+        FwTextPropType.ktptHyphenate,
+        FwTextPropType.ktptWidowOrphanControl,
+        FwTextPropType.ktptMaxLines,
+        FwTextPropType.ktptCellBorderWidth,
+        FwTextPropType.ktptCellSpacing,
+        FwTextPropType.ktptCellPadding,
+        FwTextPropType.ktptSetRowDefaults,
+        FwTextPropType.ktptRelLineHeight,
+        FwTextPropType.ktptTableRule,
+    ];
+
+    public static readonly FwTextPropType[] StringProps = [
+        FwTextPropType.ktptCharStyle,
+        FwTextPropType.ktptFontFamily,
+        FwTextPropType.ktptNamedStyle,
+        FwTextPropType.ktptBulNumTxtBef,
+        FwTextPropType.ktptBulNumTxtAft,
+        FwTextPropType.ktptFontVariations,
+        FwTextPropType.ktptParaStyle,
+        FwTextPropType.ktptTabList,
+        FwTextPropType.ktptCustomBullet,
+        FwTextPropType.ktptBulNumFontInfo,
```

Comment:
This looks like a pretty teriffying value under the hood 😱 😅

Reply:
Yeah that one and WsStyle